ord_schema/proto/reaction.proto

```diff
@@ -865,6 +869,7 @@ message ReactionProduct {
   Texture texture = 6;
   // Any features (e.g., calculated descriptors) of the product compound.
   map<string, Data> features = 7;
+  ReactionRole.ReactionRoleType reaction_role = 8;
```

Comment:
Please add a comment e.g. must be one of PRODUCT or INTERNAL_STANDARD

Reply:
I thought we were allowing any reaction role to be specified. That way, e.g., recovered starting materials could be specified with `reaction_role` `REACTANT` even though they're `ProductCompound`s. Can add a comment to help clarify this, if what I'm thinking is correct.

Reply:
Ah, that's right; a comment would be great.

Reply:
Done, thanks!